src/sentry/feedback/usecases/create_feedback.py

```diff
@@ -73,6 +73,17 @@ def fix_for_issue_platform(event_data):
     ret_event["request"] = event_data.get("request", {})
 
     ret_event["user"] = event_data.get("user", {})
+
+    if event_data.get("dist") is not None:
```

Comment:
`event_data.pop("dist", None)`

Reply:
If you want the key to continue existing if its value is `None` then this option (and the following options) won't work.